packages/client/components/MeetingNavList.tsx

```diff
@@ -7,7 +7,8 @@ const MeetingNavList = styled('ul')({
   listStyle: 'none',
   margin: 0,
   minHeight: 0, // very important! allows children to collapse for overflow
-  padding: 0
+  padding: 0,
+  overflow: 'auto'
```

Comment:
the sidebar respects the parabol logo footer

Reply:
Nice, I think this is the key change! I was trying to get the nav list to have a fixed height and make the individual phases scrollable but making the entire list scrollable works nicely. Thanks a lot for your help!